modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -290,6 +290,15 @@ TEST_P(Test_ONNX_layers, BatchNormalization3D)
     testONNXModels("batch_norm_3d");
 }
 
+TEST_P(Test_ONNX_layers, BatchNormalizationUnfused)
+{
+    if (backend == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019)
+        applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_NN_BUILDER);
+    if (backend == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH)
+        applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_NGRAPH);
```

Comment:
please remove skip tags if the test passes

Reply:
@dkurt I have tried removing skip tags, tests are getting failed for some combinations of backend and target. So should I add skip tags for those specific combinations or it is fine the way it is right now ?